planet/src/main/java/com/google/planet/servlets/ItineraryServlet.java

```diff
@@ -20,8 +20,8 @@
 import com.google.planet.data.Event;
 import com.google.planet.data.ItineraryGenerator;
 import com.google.planet.data.ItineraryItem;
-import com.google.planet.data.ItineraryOrder;
 import com.google.planet.data.ItineraryException;
+import com.google.planet.data.ItineraryOrder;
```

Comment:
Do we need this?

Reply:
ItineraryOrder has OPTIMIZED and UNOPTIMIZED which are boolean values to pass in the itinerary generator function. I used them instead of primitives (true/false) hoping to make the code more readable

Reply:
Why is the import line the only change in this file? Did you start using the enums in a previous change and it wouldn't build?

Reply:
Oh it was there before too hahah. I accidentally reordered it when resolving merge conflicts